freud/environment.pyx

```diff
@@ -137,6 +137,11 @@ cdef class BondOrder(Compute):
     def __dealloc__(self):
         del self.thisptr
 
+    @property
+    def default_query_args(self):
+        return dict(mode="nearest", r_max=self.r_max,
+                    num_neighbors=self.num_neighbors)
+
     @Compute._compute()
     def accumulate(self, box, points, orientations, query_points=None,
                    query_orientations=None, str mode="bod", nlist=None):
```

Comment:
Actually one more comment, `query_args={}` should be added as an optional argument here.

Reply:
Also why is mode typed?

Reply:
Punting this for a later rewrite to support QueryArgs.